sdk/translation/Azure.AI.Translator.Document/tsp-location1.yaml

```diff
@@ -0,0 +1,4 @@
+commit: ecf717be299977f1899bfea11a73f6c723698689
```

Comment:
I have to rename this file so as to cheat the CI to not build this SDK. The reason is that I introduced a `MultipartFormFileData` which is used as type of the properties of the customized model. CSharp generator has an issue when handling custom models indirectly used.